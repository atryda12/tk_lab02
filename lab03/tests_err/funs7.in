
int a = 0, b = 0, c = 0;

int gcd_iter(int a, int b) {
    int c = 0;

    while( a != 0 ) {
        c = a;
        a = b % a;
        b = c;
    }
}

print gcd_iter(125,100);

