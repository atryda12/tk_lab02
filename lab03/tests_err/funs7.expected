
Error: Missing return statement in function 'gcd_iter' returning int: line 12
